src/Stryker.Core/Stryker.Core/Mutators/IfStatementMutator.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Stryker.Core.Mutants;
+
+namespace Stryker.Core.Mutators
+{
+    public class IfStatementMutator : MutatorBase<IfStatementSyntax>, IMutator
+    {
+        public override IEnumerable<Mutation> ApplyMutations(IfStatementSyntax node)
```

Comment:
I'm not sure if this mutator should only target if statements. What about:

``` cs
While (person.IsAdult()) {
...
}
```
We could negate this too right? I think we can find more cases that could be worth negating. It might be better to look at the `ExpressionSyntax` node or some other Expression node type.

Reply:
I think I see it now, good idea :+1:

Reply:
See #444 for my thoughts! I think we can target `ExpressionSyntax` nodes and look in their parent nodes. If the parent is a `IfStatementSyntax` or `WhileStatementSyntax` we could mutate the `EpxressionSyntax` node :)